mesh/mesh.cpp

```diff
@@ -10431,22 +10431,44 @@ int Mesh::FindPoints(DenseMatrix &point_mat, Array<int>& elem_ids,
    return pts_found;
 }
 
-
 GeometricFactors::GeometricFactors(const Mesh *mesh, const IntegrationRule &ir,
                                    int flags)
 {
    this->mesh = mesh;
    IntRule = &ir;
    computed_factors = flags;
 
-   const GridFunction *nodes = mesh->GetNodes();
+   this->nodes = mesh->GetNodes();
```

Comment:
Not necessary -- done inside `Assemble`.

Also, maybe add an `MFEM_VERIFY` that `mesh->GetNodes()` is not `NULL`.